db/migrate/20190731092338_backfill_add_acts_as_product_to_service.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class BackfillAddActsAsProductToService < ActiveRecord::Migration
+  disable_ddl_transaction!
+
+  def change
+    Service.find_in_batches(batch_size: 200) do |records|
+      Service.where(id: records.map(&:id)).update_all act_as_product: false
```

Comment:
Let's make sure it works in Oracle as well. `update_all` doesn't go through ActiveRecord’s type-casting so the value must be supported as-is in the final SQL command.

Reply:
Ok, I will check 👍

Reply:
It probably should because I ran it locally (with a couple of services in the DB) and it didn't raise any error... but I will check the final value, just in case 😄

Reply:
It works for Oracle 💪 
![image](https://user-images.githubusercontent.com/11318903/62281005-43885380-b44d-11e9-9394-fd093cb1a2cf.png)